dcpy/connectors/socrata/publish.py

```diff
@@ -69,14 +69,43 @@ class DatasetMetadata(BaseModel):
             description: str
             tags: list[str]
             metadata: dict[str, Any]
+            privateMetadata: dict[str, Any]
 
             @classmethod
             def from_dataset_attributes(cls, attrs: md.DatasetAttributes):
                 return cls(
                     name=attrs.display_name,
                     description=attrs.description,
                     tags=attrs.tags,
-                    metadata={"rowLabel": attrs.each_row_is_a},
+                    metadata={
+                        "rowLabel": attrs.each_row_is_a,
+                        "custom_fields": {
+                            "Update": {
+                                "Update Frequency": attrs.publishing_frequency,
+                                "Automation": "Yes",
+                            }
+                        },
+                    },
+                    privateMetadata={
+                        "custom_fields": {
+                            "Legislative Compliance": {
```

Comment:
What's the background on this field?

Reply:
These are the required legislative compliance fields that you see on the dataset. 
![image](https://github.com/user-attachments/assets/2f760b74-42fa-42a3-9573-f380c75ef919)

And FYI, for the meaning of specific fields, OTI sent [this guide](https://nyco365.sharepoint.com/:x:/r/sites/NYCPLANNING/itd/edm/_layouts/15/Doc.aspx?sourcedoc=%7B842B9010-17D2-4995-8E2A-133A54C65CD3%7D&file=OTI%20Open%20Data%20Metadata%20Guide.xlsx&action=default&mobileredirect=true) over.